### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by bbarker**
I could get the existing build to work, but luckily everything just compiled after updating dependencies.

**However**, for some reason, the code does not render as HTML in either browser I tried. I'm guessing this is related to a contentType header issue, but would need to look into it more.

### CODE REVIEW

**Comment by vitaliis** [comment]
Thank you for the contribution.

Could you tell me a bit more about nix? Is it used to declare reproducible environment (tools and versions) in which given program will run? In our case we only declare a version of mill, which will run the app, correct? Although almost any other mill version will also successfully run this program, so not sure what provided nix declarations mean.

File: build.sc [hunk 1]
```diff
@@ -5,24 +5,27 @@ import mill.scalajslib.ScalaJSModule
 import mill.scalalib._
 import mill.scalalib.api.CompilationResult
 
+val projectScalaVersion = "2.13.5"
+
 //noinspection ScalaFileName
+
```
Comment (vitaliis) [inline build.sc:11]: Any reason for blank line here? Some linter or IDE rules?

**Review by vitaliis — APPROVED** [review]

**Comment by bbarker** [comment]
Hi!

> Could you tell me a bit more about nix? Is it used to declare reproducible environment (tools and versions) in which given program will run? 

Yes

> In our case we only declare a version of mill, which will run the app, correct? Although almost any other mill version will also successfully run this program, so not sure what provided nix declarations mean.

The main reason I put this in admittedly as to support possibly larger projects that build off this one, but I just updated this project to Scala3 in [another branch](https://github.com/bbarker/ScalaJS-Mill-WebApp/tree/Scala3), so it required Mill 0.9.6+ specifically. It also has a hidden dependency on `curl`, which is declared in the `default.nix` file. As is usually the case, I find that people that use nix will just find these and make use of them if available, without any expectation of them being maintained (one would hope). Others could ignore them

Regarding the Scala 3 branch, I can make a pull request off that one instead (I didnt' actually need to change the source code, though probably could make some adjustments). 

If you prefer me to leave off the nix files that's fine.

**Comment by vitaliis** [comment]
I am happy to learn that such tool exists and will merge nix-declarations changes. Thanks for contribution once again! And feel free to open another pull request if you have some more suggestions. I also just enabled discussions, so feel free to post there :)

File: build.sc (hunk 1 above)
Comment (bbarker) [reply build.sc:11]: That might have been an accident on my part

**Comment by vitaliis** [comment]
@bbarker, I simplified the build file a bit. Anticipated shared module wasn't useful, so removed for now.
Just FYI in case you want to build up or open further MRs/PRs.

**Pull request merged**